site/docs/concepts/config-policies.md

```diff
@@ -86,6 +87,21 @@ config[{"write": decision}] {
 }
 ```
 
+### Versioning 
+
+Policy versioning is done by defining a `version` variable inside the `adminconfig` package. 
+
+charts/fybrik/files/adminconfig/version.rego defines the version for the policies:
+
+```
+package adminconfig
+
+version := "0.0.1"
```

Comment:
Does this mean that if we have lots of files containing rego policies, and each file has multiple policies in it ... they all have the same version?

Won't it be difficult to know what changed between the versions?  Is there a reason not to have a version in each policy?